### PR TITLE
Rabbitmq integration tests.

### LINKED ISSUE #5605: latest edge-21.1.3 breaks tcp mTLS in  RabbitMQ
### What is the issue?
Using linkerd stable-2.9.0 to mesh a RabbitMQ cluster mTLSed endpoints works perfectly out of the box (no port annotations), upgrading to edge-21.1.3 causes RabbitMQ to break 
linkerd now throws this `linkerd_proxy_http::detect: Could not detect protocol read=9` 

### How can it be reproduced?
install edge-21.1.3 to a rabbitMQ 3 node cluster  check the logs 

### Logs, error output, etc
linkerd-proxy 
```log
[ 21.805559s] DEBUG ThreadId(01) outbound:accept{peer.addr=10.0.101.183:48423 target.addr=10.0.100.168:4369}: linkerd_proxy_http::detect: Could not detect protocol read=9
  [ 21.805598s] DEBUG ThreadId(01) outbound:accept{peer.addr=10.0.101.183:48423 target.addr=10.0.100.168:4369}: linkerd_detect: Detected protocol=None elapsed=6.990899484s
  [ 21.805615s] DEBUG ThreadId(01) outbound:accept{peer.addr=10.0.101.183:48423 target.addr=10.0.100.168:4369}: linkerd_cache: Caching new service target=(None, Logical { orig_dst: 10.0.100.168:4369, protocol: (), profile: Some(..) })
  [ 21.805718s] DEBUG ThreadId(01) outbound:accept{peer.addr=10.0.101.183:48423 target.addr=10.0.100.168:4369}:tcp: linkerd_tls::client: Initiating TLS connection target.id="rabbitmq.default.serviceaccount.identity.linkerd.cluster.local"
  [ 21.805735s] DEBUG ThreadId(01) outbound:accept{peer.addr=10.0.101.183:48423 target.addr=10.0.100.168:4369}:tcp: linkerd_proxy_transport::connect: Connecting peer.addr=10.0.100.168:4369
  [ 21.806406s] DEBUG ThreadId(01) outbound:accept{peer.addr=10.0.101.183:48423 target.addr=10.0.100.168:4369}:tcp: linkerd_proxy_transport::connect: Connected local.addr=10.0.101.183:44588 keepalive=Some(10s)
  [ 21.808881s] DEBUG ThreadId(01) outbound:accept{peer.addr=10.0.101.183:48423 target.addr=10.0.100.168:4369}:tcp: linkerd_proxy_transport::metrics: client connection open
  [ 21.809905s] INFO ThreadId(01) outbound:accept{peer.addr=10.0.101.183:48423 target.addr=10.0.100.168:4369}: linkerd_app_core::serve: Connection closed error=server: Transport endpoint is not connected (os error 107)
  [ 21.809935s] DEBUG ThreadId(01) evict{target=(None, Logical { orig_dst: 10.0.100.168:4369, protocol: (), profile: Some(..) })}: linkerd_cache: Awaiting idleness
  [ 21.809943s] DEBUG ThreadId(01) evict{target=Accept { orig_dst: 10.0.100.168:4369, protocol: () }}: linkerd_cache: Awaiting idleness
  [ 24.578095s] DEBUG ThreadId(02) daemon:admin{listen.addr=0.0.0.0:4191}:accept{peer.addr=10.0.101.161:34884 target.addr=10.0.101.183:4191}: linkerd_tls::server: Peeked bytes from TCP stream sz=119
  [ 24.578307s] DEBUG ThreadId(02) daemon:admin{listen.addr=0.0.0.0:4191}:accept{peer.addr=10.0.101.161:34884 target.addr=10.0.101.183:4191}: linkerd_app_core::serve: Connection closed
  [ 24.816187s] DEBUG ThreadId(01) dst: linkerd_dns: resolve_srv name=linkerd-dst-headless.linkerd.svc.cluster.local
  [ 24.819857s] DEBUG ThreadId(01) dst: linkerd_dns: addrs=[10.0.100.63:8086]
  [ 24.819888s] DEBUG ThreadId(01) dst: linkerd_proxy_dns_resolve: addrs=[10.0.100.63:8086]
  [ 26.810831s] DEBUG ThreadId(01) evict{target=(None, Logical { orig_dst: 10.0.100.168:4369, protocol: (), profile: Some(..) })}: linkerd_cache: Cache entry dropped
  [ 28.066414s] DEBUG ThreadId(02) daemon:admin{listen.addr=0.0.0.0:4191}:accept{peer.addr=10.0.101.161:34936 target.addr=10.0.101.183:4191}: linkerd_tls::server: Peeked bytes from TCP stream sz=120
  [ 28.066578s] DEBUG ThreadId(02) daemon:admin{listen.addr=0.0.0.0:4191}:accept{peer.addr=10.0.101.161:34936 target.addr=10.0.101.183:4191}: linkerd_app_core::serve: Connection closed
  [ 31.489621s] DEBUG ThreadId(01) outbound:accept{peer.addr=10.0.101.183:58667 target.addr=10.0.100.168:4369}: linkerd_proxy_http::detect: Could not detect protocol read=9
  [ 31.489656s] DEBUG ThreadId(01) outbound:accept{peer.addr=10.0.101.183:58667 target.addr=10.0.100.168:4369}: linkerd_detect: Detected protocol=None elapsed=7.000235845s
  [ 31.489665s] DEBUG ThreadId(01) outbound:accept{peer.addr=10.0.101.183:58667 target.addr=10.0.100.168:4369}: linkerd_cache: Caching new service target=(None, Logical { orig_dst: 10.0.100.168:4369, protocol: (), profile: Some(..) })
  [ 31.489744s] DEBUG ThreadId(01) outbound:accept{peer.addr=10.0.101.183:58667 target.addr=10.0.100.168:4369}:tcp: linkerd_tls::client: Initiating TLS connection target.id="rabbitmq.default.serviceaccount.identity.linkerd.cluster.local"
  [ 31.489755s] DEBUG ThreadId(01) outbound:accept{peer.addr=10.0.101.183:58667 target.addr=10.0.100.168:4369}:tcp: linkerd_proxy_transport::connect: Connecting peer.addr=10.0.100.168:4369
  [ 31.490529s] DEBUG ThreadId(01) outbound:accept{peer.addr=10.0.101.183:58667 target.addr=10.0.100.168:4369}:tcp: linkerd_proxy_transport::connect: Connected local.addr=10.0.101.183:44692 keepalive=Some(10s)
  [ 31.491475s] DEBUG ThreadId(01) outbound:accept{peer.addr=10.0.101.183:58667 target.addr=10.0.100.168:4369}:tcp: linkerd_proxy_transport::metrics: client connection open
  [ 31.492797s] INFO ThreadId(01) outbound:accept{peer.addr=10.0.101.183:58667 target.addr=10.0.100.168:4369}: linkerd_app_core::serve: Connection closed error=server: Transport endpoint is not connected (os error 107)
  [ 31.492832s] DEBUG ThreadId(01) evict{target=(None, Logical { orig_dst: 10.0.100.168:4369, protocol: (), profile: Some(..) })}: linkerd_cache: Awaiting idleness
  [ 34.578075s] DEBUG ThreadId(02) daemon:admin{listen.addr=0.0.0.0:4191}:accept{peer.addr=10.0.101.161:34988 target.addr=10.0.101.183:4191}: linkerd_tls::server: Peeked bytes from TCP stream sz=119
  [ 34.578404s] DEBUG ThreadId(02) daemon:admin{listen.addr=0.0.0.0:4191}:accept{peer.addr=10.0.101.161:34988 target.addr=10.0.101.183:4191}: linkerd_app_core::serve: Connection closed
  [ 36.494443s] DEBUG ThreadId(01) evict{target=(None, Logical { orig_dst: 10.0.100.168:4369, protocol: (), profile: Some(..) })}: linkerd_cache: Cache entry dropped
  [ 38.066389s] DEBUG ThreadId(02) daemon:admin{listen.addr=0.0.0.0:4191}:accept{peer.addr=10.0.101.161:35040 target.addr=10.0.101.183:4191}: linkerd_tls::server: Peeked bytes from TCP stream sz=120
  [ 38.066579s] DEBUG ThreadId(02) daemon:admin{listen.addr=0.0.0.0:4191}:accept{peer.addr=10.0.101.161:35040 target.addr=10.0.101.183:4191}: linkerd_app_core::serve: Connection closed
  [ 38.539670s] DEBUG ThreadId(01) outbound:accept{peer.addr=10.0.101.183:43655 target.addr=10.0.100.168:4369}: linkerd_proxy_http::detect: Could not detect protocol read=9
  [ 38.539729s] DEBUG ThreadId(01) outbound:accept{peer.addr=10.0.101.183:43655 target.addr=10.0.100.168:4369}: linkerd_detect: Detected protocol=None elapsed=7.000989191s
  [ 38.539739s] DEBUG ThreadId(01) outbound:accept{peer.addr=10.0.101.183:43655 target.addr=10.0.100.168:4369}: linkerd_cache: Caching new service target=(None, Logical { orig_dst: 10.0.100.168:4369, protocol: (), profile: Some(..) })
  [ 38.539835s] DEBUG ThreadId(01) outbound:accept{peer.addr=10.0.101.183:43655 target.addr=10.0.100.168:4369}:tcp: linkerd_tls::client: Initiating TLS connection target.id="rabbitmq.default.serviceaccount.identity.linkerd.cluster.local"
```
RabbitMQ 
```log
 ## ## RabbitMQ 3.7.3. Copyright (C) 2007-2018 Pivotal Software, Inc.
  ########## Licensed under the MPL. See http://www.rabbitmq.com/
  ###### ##
  ########## Logs: <stdout>

  Starting broker...
  2021-01-25 14:49:07.540 [info] <0.264.0>
  node : rabbit@rabbitmq-2.rabbitmq.default.svc.cluster.local
  home dir : /var/lib/rabbitmq
  config file(s) : /etc/rabbitmq/rabbitmq.conf
  cookie hash : G9xFiGrZ1fE/RXO9dM2I7Q==
  log(s) : <stdout>
  database dir : /var/lib/rabbitmq/mnesia/rabbit@rabbitmq-2.rabbitmq.default.svc.cluster.local
  2021-01-25 14:49:08.108 [info] <0.270.0> Memory high watermark set to 1953 MiB (2048000000 bytes) of 3072 MiB (3221225472 bytes) total
  2021-01-25 14:49:08.112 [info] <0.272.0> Enabling free disk space monitoring
  2021-01-25 14:49:08.112 [info] <0.272.0> Disk free limit set to 50MB
  2021-01-25 14:49:08.115 [info] <0.274.0> Limiting to approx 1048476 file handles (943626 sockets)
  2021-01-25 14:49:08.115 [info] <0.275.0> FHC read buffering: OFF
  2021-01-25 14:49:08.115 [info] <0.275.0> FHC write buffering: ON
  2021-01-25 14:49:29.137 [info] <0.264.0> Waiting for Mnesia tables for 30000 ms, 9 retries left
  2021-01-25 14:50:04.950 [info] <0.33.0> Application lager started on node 'rabbit@rabbitmq-0.rabbitmq.default.svc.cluster.local'
  14:49:59.138 [warning] <0.264.0> Error while waiting for Mnesia tables: {timeout_waiting_for_tables,[rabbit_user,rabbit_user_permission,rabbit_topic_permission,rabbit_vhost,rabbit_durable_route,rabbit_durable_exchange,rabbit_runtime_parameters,rabbit_durable_queue]}
  rabbitmq 2021-01-25 14:49:59.138 [info] <0.264.0> Waiting for Mnesia tables for 30000 ms, 8 retries left
  14:50:27.910 [info] <0.33.0> Application xmerl started on node 'rabbit@rabbitmq-0.rabbitmq.default.svc.cluster.local'
  rabbitmq 2021-01-25 14:50:27.911 [info] <0.33.0> Application os_mon started on node 'rabbit@rabbitmq-0.rabbitmq.default.svc.cluster.local'
  2021-01-25 14:50:27.911 [info] <0.33.0> Application jsx started on node 'rabbit@rabbitmq-0.rabbitmq.default.svc.cluster.local'
  2021-01-25 14:50:29.139 [warning] <0.264.0> Error while waiting for Mnesia tables: {timeout_waiting_for_tables,[rabbit_user,rabbit_user_permission,rabbit_topic_permission,rabbit_vhost,rabbit_durable_route,rabbit_durable_exchange,rabbit_runtime_parameters,rabbit_durable_queue]}
  rabbitmq 2021-01-25 14:50:29.139 [info] <0.264.0> Waiting for Mnesia tables for 30000 ms, 7 retries left
  [info] <0.33.0> Application inets started on node 'rabbit@rabbitmq-0.rabbitmq.default.svc.cluster.local'
  2021-01-25 14:50:55.958 [info] <0.33.0> Application mnesia started on node 'rabbit@rabbitmq-0.rabbitmq.default.svc.cluster.local'
  2021-01-25 14:50:55.958 [info] <0.33.0> Application amqp10_common started on node 'rabbit@rabbitmq-0.rabbitmq.default.svc.cluster.local'
  2021-01-25 14:50:55.958 [info] <0.33.0> Application crypto started on node 'rabbit@rabbitmq-0.rabbitmq.default.svc.cluster.local'
  2021-01-25 14:50:55.959 [info] <0.33.0> Application cowlib started on node 'rabbit@rabbitmq-0.rabbitmq.default.svc.cluster.local'
  2021-01-25 14:50:55.959 [info] <0.33.0> Application asn1 started on node 'rabbit@rabbitmq-0.rabbitmq.default.svc.cluster.local'
  2021-01-25 14:50:55.959 [info] <0.33.0> Application public_key started on node 'rabbit@rabbitmq-0.rabbitmq.default.svc.cluster.local'
  2021-01-25 14:50:55.960 [info] <0.33.0> Application ssl started on node 'rabbit@rabbitmq-0.rabbitmq.default.svc.cluster.local'
  2021-01-25 14:50:55.967 [info] <0.33.0> Application amqp10_client started on node 'rabbit@rabbitmq-0.rabbitmq.default.svc.cluster.local'
  2021-01-25 14:50:55.967 [info] <0.33.0> Application ranch started on node 'rabbit@rabbitmq-0.rabbitmq.default.svc.cluster.local'
  2021-01-25 14:50:55.973 [info] <0.33.0> Application cowboy started on node 'rabbit@rabbitmq-0.rabbitmq.default.svc.cluster.local'
  2021-01-25 14:50:55.973 [info] <0.33.0> Application ranch_proxy_protocol started on node 'rabbit@rabbitmq-0.rabbitmq.default.svc.cluster.local'
  2021-01-25 14:50:55.973 [info] <0.33.0> Application recon started on node 'rabbit@rabbitmq-0.rabbitmq.default.svc.cluster.local'
  2021-01-25 14:50:55.973 [info] <0.33.0> Application rabbit_common started on node 'rabbit@rabbitmq-0.rabbitmq.default.svc.cluster.local'
  2021-01-25 14:50:55.978 [info] <0.33.0> Application amqp_client started on node 'rabbit@rabbitmq-0.rabbitmq.default.svc.cluster.local'
  2021-01-25 14:50:55.979 [info] <0.314.0>
  Starting RabbitMQ 3.7.3 on Erlang 20.1.7
  Copyright (C) 2007-2018 Pivotal Software, Inc.
  Licensed under the MPL. See http://www.rabbitmq.com/

  ## ##
  ## ## RabbitMQ 3.7.3. Copyright (C) 2007-2018 Pivotal Software, Inc.
  ########## Licensed under the MPL. See http://www.rabbitmq.com/
  ###### ##
  ########## Logs: <stdout>

  Starting broker...
  2021-01-25 14:50:55.980 [info] <0.314.0>
  node : rabbit@rabbitmq-0.rabbitmq.default.svc.cluster.local
  home dir : /var/lib/rabbitmq
  config file(s) : /etc/rabbitmq/rabbitmq.conf
  cookie hash : G9xFiGrZ1fE/RXO9dM2I7Q==
  log(s) : <stdout>
  database dir : /var/lib/rabbitmq/mnesia/rabbit@rabbitmq-0.rabbitmq.default.svc.cluster.local
  2021-01-25 14:50:56.720 [info] <0.320.0> Memory high watermark set to 1953 MiB (2048000000 bytes) of 3072 MiB (3221225472 bytes) total
  2021-01-25 14:50:56.724 [info] <0.322.0> Enabling free disk space monitoring
  2021-01-25 14:50:56.724 [info] <0.322.0> Disk free limit set to 50MB
  2021-01-25 14:50:56.727 [info] <0.325.0> FHC read buffering: OFF
  2021-01-25 14:50:56.728 [info] <0.325.0> FHC write buffering: ON
  2021-01-25 14:50:56.728 [info] <0.324.0> Limiting to approx 1048476 file handles (943626 sockets)
  2021-01-25 14:50:59.140 [warning] <0.264.0> Error while waiting for Mnesia tables: {timeout_waiting_for_tables,[rabbit_user,rabbit_user_permission,rabbit_topic_permission,rabbit_vhost,rabbit_durable_route,rabbit_durable_exchange,rabbit_runtime_parameters,rabbit_durable_queue]}
  2021-01-25 14:50:59.140 [info] <0.264.0> Waiting for Mnesia tables for 30000 ms, 6 retries left
  2021-01-25 14:51:07.604 [warning] <0.268.0> Error while waiting for Mnesia tables: {timeout_waiting_for_tables,[rabbit_user,rabbit_user_permission,rabbit_topic_permission,rabbit_vhost,rabbit_durable_route,rabbit_durable_exchange,rabbit_runtime_parameters,rabbit_durable_queue]}
  2021-01-25 14:51:07.604 [info] <0.268.0> Waiting for Mnesia tables for 30000 ms, 8 retries left
  2021-01-25 14:51:29.141 [warning] <0.264.0> Error while waiting for Mnesia tables: {timeout_waiting_for_tables,[rabbit_user,rabbit_user_permission,rabbit_topic_permission,rabbit_vhost,rabbit_durable_route,rabbit_durable_exchange,rabbit_runtime_parameters,rabbit_durable_queue]}
  2021-01-25 14:51:29.141 [info] <0.264.0> Waiting for Mnesia tables for 30000 ms, 5 retries left
  2021-01-25 14:51:37.605 [warning] <0.268.0> Error while waiting for Mnesia tables: {timeout_waiting_for_tables,[rabbit_user,rabbit_user_permission,rabbit_topic_permission,rabbit_vhost,rabbit_durable_route,rabbit_durable_exchange,rabbit_runtime_parameters,rabbit_durable_queue]}
  2021-01-25 14:51:37.605 [info] <0.268.0> Waiting for Mnesia tables for 30000 ms, 7 retries left
  2021-01-25 14:51:52.767 [info] <0.314.0> Waiting for Mnesia tables for 30000 ms, 9 retries left
  2021-01-25 14:51:52.831 [info] <0.314.0> Waiting for Mnesia tables for 30000 ms, 9 retries left
```

### Environment

- Kubernetes Version:
- 1.16
- Cluster Environment: (GKE, AKS, kops, ...)
- EKS

- Linkerd version:
- edge-21.1.3

### PR DESCRIPTION
**Pull request by barkardk**
Subject
   Add integration tests for external resources
Problem
  linkerd changes sometimes causes regressions on external components that users have installed in their stacks such as rabbitmq. 
Solution
New integration test,  adds functionality to install additional components and run basic test on them  to make sure linkerd  changes do not have an adverse effect
  This change will include the following steps
 Deploy a rabbitmq server
 Deploy a rabbitmq client, the code for that client is hosted at https://github.com/barkardk/integration
 Use a golden file for output verification
 Uses the test suite to inject linkerd into the deployments 
 Client then creates a queue, a message and consumes said message and sends a log output.
 The linkerd testutil compares the output from the client with a golden file

Fixes #5605

Signed-off-by: Kristin Barkardottir <kristin.barkardottir@netapp.com>

### CODE REVIEW

**Review by adleong — COMMENTED** [review]
Thanks, @barkardk!  This looks like the right approach to me!  I'll do some more testing with this later.

I don't think we need to add an `--external-resources` flag to control this test.  It seems safe enough to just always run this test as part of our regular integration suite.

**Review by skeggi — APPROVED** [review]
Nice and clean, so far.

(Sorry this was supposed to be a comment only, not approval. However it seems that I do not have rights to revoke. But as this is a draft only it should not matter)

File: test/integration/externalresources/testdata/rabbitmq-server.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      containers:
+        - image: rabbitmq:3-management
+          name: rabbitmq
+          livenessProbe:
+            exec:
+              command:
+                - rabbitmqctl
+                - status
+            failureThreshold: 6
+            initialDelaySeconds: 120
```
Comment (skeggi) [inline test/integration/externalresources/testdata/rabbitmq-server.yaml:26]: Please review length of delay

File: test/integration/externalresources/testdata/rabbitmq-server.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            periodSeconds: 10
+            successThreshold: 1
+            timeoutSeconds: 5
+          readinessProbe:
+            exec:
+              command:
+                - rabbitmqctl
+                - status
+            failureThreshold: 6
+            initialDelaySeconds: 120
```
Comment (skeggi) [inline test/integration/externalresources/testdata/rabbitmq-server.yaml:36]: See above

File: test/integration/externalresources/testdata/rabbitmq-server.yaml (hunk 1 above)
Comment (barkardk) [reply test/integration/externalresources/testdata/rabbitmq-server.yaml:26]: I changed this to 10 seconds

**Comment by Pothulapati** [comment]
@barkardk Hey, Thanks for taking this up.

Can you resolve the current merge conflicts? :)

**Comment by barkardk** [comment]
@Pothulapati  done

**Review by adleong — COMMENTED** [review]
This is great!!!

I'd recommend using a fixed tag instead of `latest` in `ghcr.io/barkardk/rabbitmq-client:latest` so that an update to this docker image doesn't unexpectedly change or break this test.  Other than that, I think this is good to go.

**Comment by barkardk** [comment]
@adleong  I have set both rabbitmq server and the client to use fixed tagged versions. 
The rabbitmq-client repository also has a build pipeline that releases new stable versions via tags to protect the images

**Review by adleong — APPROVED** [review]
Awesome!  Thank you so much for this, @barkardk!

File: bin/_test-helpers.sh [hunk 3]
```diff
@@ -528,6 +529,12 @@ run_cluster-domain_test() {
   run_test "$test_directory/install_test.go" --cluster-domain='custom.domain'
 }
 
+# wrapper to implement external tests
+run_externalresources_test(){
```
Comment (Pothulapati) [inline bin/_test-helpers.sh:533]: nit: How about calling this `external-resources` as we follow those semantics everywhere?

File: go.mod [hunk 4]
```diff
@@ -44,6 +44,7 @@ require (
 	golang.org/x/tools v0.0.0-20200616133436-c1934b75d054
 	google.golang.org/grpc v1.31.1
 	google.golang.org/protobuf v1.24.0
+	gotest.tools/gotestsum v0.4.2 // indirect
```
Comment (Pothulapati) [inline go.mod:47]: Wondering why this is added, as this is not being used anywhere. Can you do `go mod tidy` for the go tool to clean it up?

**Review by Pothulapati — COMMENTED** [review]
Looks good overall, except for the below nits :)

**Comment by kleimkuhler** [comment]
@barkardk We have an issue open about pods have restart counts, so I don't think the CI failures are related to your change. Once we get that solved, we'll leave a comment here and they should start passing again.

**Comment by kleimkuhler** [comment]
@barkardk If you merge in the latest `main`, it should pick up the fix causing CI to fail right now.

**Comment by barkardk** [comment]
@kleimkuhler I seem to still have the same issue after merge

**Comment by barkardk** [comment]
@kleimkuhler  I still see various timeouts in the checks

**Comment by kleimkuhler** [comment]
@barkardk Hm yea sorry about this. This is happening for a few other PRs as well. We'll see what's going on and keep you updated; hopefully you'll be able to merge in a fix later today.

**Comment by barkardk** [comment]
@Pothulapati are you good with the changes ?

**Comment by Pothulapati** [comment]
@barkardk Can you also add the `external-resources` integration test to [release and integration workflows in CI](https://github.com/linkerd/linkerd2/tree/main/.github/workflows) sot that they can be ran automatically?

**Comment by barkardk** [comment]
@Pothulapati  as per suggestion from @adleong  these tests are part of the default suite , they do not need to be enabled specifically. 
[https://github.com/linkerd/linkerd2/blob/7c574d3cad05f11e2fad1c5aaf8be24dfc38c679/bin/_test-helpers.sh#L10-L11](_test-helpers.sh)

**Comment by Pothulapati** [comment]
@barkardk The link you specified seems to be broken.

Also, Adding to the default suite, does not actually enable them in CI. [As you can see in the workflows, we list and run them individually](https://github.com/linkerd/linkerd2/blob/main/.github/workflows/integration_tests.yml#L100), Once you update this both in the `integration_tests.yml` and `release.yml`, You should be able to see the integration test in this PR's Github Action checks :)

**Comment by barkardk** [comment]
@Pothulapati alright. I have added the new test to `integration_tests.yml` and `release.yml`

**Review by Pothulapati — APPROVED** [review]
LGTM, Thanks for taking this up.

**Comment by Pothulapati** [comment]
Just merged this. 

Sorry for not merging it right after it was approved. We were waiting for `2.10` to be released. So, As to make sure there is no flakiness.

**Pull request merged**